src/machinetalk/application/commandbase.cpp

```diff
@@ -353,7 +353,7 @@ void CommandBase::sendEmcAxisUnhome(Container &tx)
 
 void CommandBase::sendShutdown(Container &tx)
 {
-    sendCommandMessage(MT_SHUTDOWN, tx);
+    sendCommandMessage(MT_LAUNCHER_SHUTDOWN, tx);
```

Comment:
``MT_SHUTDOWN`` is for shutting down the Machinekit application.
I would like to use ``MT_LAUNCHER_SHUTDOWN`` to shutdown the machine.

mkwrapper.py:
```        elif self.rx.type == MT_SHUTDOWN:
                self.send_command_msg(identity, MT_CONFIRM_SHUTDOWN)
                self.stop()  # trigger the shutdown event
            
            elif self.rx.type == MT_LAUNCHER_SHUTDOWN:
                try:
                    systemBus = dbus.SystemBus()
                    ckService = systemBus.get_object('org.freedesktop.ConsoleKit',
                                                     '/org/freedesktop/ConsoleKit/Manager')
                    ckInterface = dbus.Interface(ckService, 'org.freedesktop.ConsoleKit.Manager')
                    stopMethod = ckInterface.get_dbus_method("Stop")
                    stopMethod()
                    print "LAUNCHER_SHUTDOWN PASS"
                except:
                    print "LAUNCHER_SHUTDOWN ERROR"
```